### PR TITLE
[113][116][119] Implement resume view and edit page

### PR DESCRIPTION
**Pull request by miello**
## What did you do

- Implement resume edit page
- Implement resume add and delete functionality

## Demo

- Login as actor and go to https://dev.modela.miello.dev/profile/resume

image

image

## Limitation

- Upload file is not available yet so resume url will be a placeholder

### CODE REVIEW

**Comment by PKhing** [comment]
image
Change card variant

**Comment by PKhing** [comment]
Now all the space is clickable should we change to just clickable on text? 
image

**Comment by PKhing** [comment]
image
change button variant to match with the design ?

**Comment by PKhing** [comment]
should this text be like "download resume" or "view resume" ?
image

File: apps/web/src/modules/profile/edit/actor/resume/components/ResumeSlot/hooks/useResumeSlot/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { useCallback, useState } from 'react'
+
+const useResumeSlot = (
+  isFirst: boolean,
+  resumeId: number,
+  handleCancel: (resumeId: number) => void,
+) => {
+  const [isEdit, setEdit] = useState(!!isFirst)
```
Comment (PKhing) [inline apps/web/src/modules/profile/edit/actor/resume/components/ResumeSlot/hooks/useResumeSlot/index.tsx:8]: why `!!isFirst` when it is not optional?

Comment (miello) [reply apps/web/src/modules/profile/edit/actor/resume/components/ResumeSlot/hooks/useResumeSlot/index.tsx:8]: Copy and paste syndrome 😂

**Review by PKhing — APPROVED** [review]

**Pull request merged**